doc/rados/operations/stretch-mode.rst

```diff
@@ -297,30 +302,29 @@ To exit stretch mode, run the following command:
 
 .. describe:: {crush_rule}
 
-   The CRUSH rule that the user wants all pools to move back to. If this
-   is not specified, the pools will move back to the default CRUSH rule.
+   The CRUSH rule to now use for all pools. If this
```

Comment:
There's got to be something clearer than "to now use". I don't know what it is off the top of my head, but I'm going to keep thinking about it.